commit c7d3332a736228a9e066f395adbf9c9ef827ea6c (refs/changes/39/4939/1)
Author: Marsell Kukuljevic <marsell@joyent.com>
Date:   2018-10-11T07:55:15+13:00 (12 months ago)
    
    TRITON-800 "Bhyve Max Vcpus" should be ignored if "Bhyve Capable" is not true

diff --git a/package.json b/package.json
index 27b790d..c764497 100644
--- a/package.json
+++ b/package.json
@@ -8,7 +8,7 @@
     "assert-plus": "~1.0.0",
     "bunyan": "1.8.5",
     "libuuid": "0.2.1",
-    "joyent-schemas": "git+https://github.com/joyent/schemas.git#9f0791091868f449c8933902b556652e6c24875c",
+    "joyent-schemas": "git+https://github.com/joyent/schemas.git#09cebfca94169601568fec60f7a6144130f7fd31",
     "jsprim": "1.3.1",
     "vasync": "~1.6.4"
   },
